front/src/Phaser/Game/GameScene.ts

```diff
@@ -214,7 +215,7 @@ export class GameScene extends DirtyScene {
         this.Terrains = [];
         this.groups = new Map<number, Sprite>();
         this.instance = room.getInstance();
-        room.getContactPage();
+        contactPageStore.set(room.contactPage);
```

Comment:
If you set the store in the constructor, preloading of other maps will set the store for you.
You should really call this store when we enter the room (so probably in the "create" method)